src/pages/intro/cdn.md

```diff
@@ -79,6 +79,80 @@ import '@ionic/react/css/flex-utils.css';
 import '@ionic/react/css/display.css';
 ```
 
+
+## Ionic + Vue
+
+To add Ionic Framework to an existing Vue project, install the `@ionic/vue` and `@ionic/vue-router` packages.
+
+```shell
+$ npm install @ionic/vue @ionic/vue-router
+```
+
+After that, you will need to install the `IonicVue` plugin in your Vue app.
+
+**main.js**
+```javascript
+import { IonicVue } from '@ionic/vue';
+
+import App from './App.vue'
+import router from './router';
+
+const app = createApp(App)
+  .use(IonicVue)
+  .use(router);
+  
+router.isReady().then(() => {
+  app.mount('#app');
+});
+```
+
+Be sure to mount your app once `router.isReady()` has resolved.
+
+### Routing
+
+When setting up routing in your Vue app, you will need to import your dependencies from `@ionic/vue-router` instead of `vue-router`.
+
+**router/index.js**
+
+```javascript
+import { createRouter, createWebHistory } from '@ionic/vue-router';
+
+const routes = [
+  // routes go here
+]
+
+const router = createRouter({
+  history: createWebHistory(process.env.BASE_URL),
+  routes
+})
+
+export default router;
+```
+
+### CSS
+
+To include the necessary CSS in a Vue project, add the following to your `main.js` file.
+
+```javascript
+/* Core CSS required for Ionic components to work properly */
+import '@ionic/react/css/core.css';
+
+/* Basic CSS for apps built with Ionic */
+import '@ionic/react/css/normalize.css';
+import '@ionic/react/css/structure.css';
+import '@ionic/react/css/typography.css';
+
+/* Optional CSS utils that can be commented out */
+import '@ionic/react/css/padding.css';
+import '@ionic/react/css/float-elements.css';
+import '@ionic/react/css/text-alignment.css';
+import '@ionic/react/css/text-transformation.css';
+import '@ionic/react/css/flex-utils.css';
+import '@ionic/react/css/display.css';
+```
```

Comment:
```suggestion
```javascript
/* Core CSS required for Ionic components to work properly */
import '@ionic/vue/css/core.css';

/* Basic CSS for apps built with Ionic */
import '@ionic/vue/css/normalize.css';
import '@ionic/vue/css/structure.css';
import '@ionic/vue/css/typography.css';

/* Optional CSS utils that can be commented out */
import '@ionic/vue/css/padding.css';
import '@ionic/vue/css/float-elements.css';
import '@ionic/vue/css/text-alignment.css';
import '@ionic/vue/css/text-transformation.css';
import '@ionic/vue/css/flex-utils.css';
import '@ionic/vue/css/display.css';
```
```

Reply:
Good catch - thank you!